instrumentation/opentelemetry_grpcbox/src/otel_grpcbox_interceptor.erl

```diff
@@ -52,14 +57,16 @@ recv_msg(#{client_stream := ClientStream}, Streamer, Input) ->
 
 unary(Ctx, Message, _ServerInfo=#{full_method := FullMethod}, Handler) ->
     otel_ctx_from_ctx(Ctx),
-    ?with_span(FullMethod, #{}, fun(_) ->
+    ?with_span(FullMethod, #{kind => ?SPAN_KIND_SERVER}, fun(_) ->
+                                        ?set_attribute(?RPC_SYSTEM, ?RPC_SYSTEM_GRPC),
```

Comment:
Actually, another issue, these can be set in the start opts along with kind:

```
#{kind => ?SPAN_KIND_SERVER
  attributes => #{?RPC_SYSTEM => ?RPC_SYSTEM_GRPC}}
```

Reply:
that i can fix... standby!

Reply:
ok, fixed (and pushed).